client/util/util_test.go

```diff
@@ -27,3 +27,14 @@ func TestLoadSignerBinaryPath(t *testing.T) {
 		t.Errorf("Expected path is %q, got: %q", want, path)
 	}
 }
+
+func TestLoadSignerBinaryPathHome(t *testing.T) {
+	path, err := LoadSignerBinaryPath("./test_data/certificate_config/homeExpansion.json")
+	if err != nil {
+		t.Errorf("LoadSignerBinaryPath error: %q", err)
+	}
+	want := "/Users/angelahao/ecp/signer"
```

Comment:
This will only work on your machine.

Instead, you could try this:
```suggestion
	want := homeDir + /ecp/signer"
```

I'll leave the actual implementation of this to you, but it is similar to the code you've written